src/main/java/Task.java

```diff
@@ -0,0 +1,38 @@
+public class Task {
+    private String name;
+    private boolean isDone;
+
+    public Task() {
+        this.name = "placeholder";
+        this.isDone = false;
+    }
```

Comment:
You can consider using the keyword this to shorten the code and improve readability. For example : 
```suggestion
    public Task() {
        this("placeholder");
    }
```

Reply:
Implemented, thanks for the suggestion